### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by MaiSan9999** [swishs-client-mod-plugins/sperm-bank#1]
There's way to convert holy notes notes.json for sperm bank config? like just copy paste somewhere?

**Comment by Swishilicous** [comment]
yeah hit the help icon in the modal then click the import button and then click powercord or holy notes i forgot what i put
